apps/mark-scan/frontend/src/lib/assistive_technology.test.tsx

```diff
@@ -117,3 +113,51 @@ it('accessible controller handling works', async () => {
   expect(screen.getButton(/Okay/i)).toHaveFocus();
   await advanceTimersAndPromises();
 });
+
+it('auto-focuses "next" button on contest screen after voting', async () => {
+  const hardware = MemoryHardware.buildStandard();
+  apiMock.expectGetMachineConfig();
+  apiMock.expectGetElectionDefinition(electionGeneralDefinition);
+  apiMock.expectGetElectionState({
+    precinctSelection: ALL_PRECINCTS_SELECTION,
+    pollsState: 'polls_open',
+  });
+  mockOf(apiMock.mockApiClient.isPatDeviceConnected).mockResolvedValue(true);
+
+  render(
+    <App
+      hardware={hardware}
+      apiClient={apiMock.mockApiClient}
+      reload={jest.fn()}
+    />
+  );
+  await advanceTimersAndPromises();
+  // Start voter session
+  apiMock.setAuthStatusCardlessVoterLoggedIn({
+    ballotStyleId: '12',
+    precinctId: '23',
+  });
+
+  userEvent.click(await screen.findButton('Start Voting'));
+
+  await screen.findByText(contest0.title);
+
+  // Confirm first contest only has 1 seat
+  expect(contest0.seats).toEqual(1);
+
+  // Test navigation by accessible controller keyboard event interface
+  userEvent.keyboard('1');
+  expect(getActiveElement()).toHaveTextContent(contest0candidate0.name);
+  userEvent.keyboard('1');
+  expect(getActiveElement()).toHaveTextContent(contest0candidate1.name);
+
+  // select candidate
+  userEvent.keyboard('2');
+  await screen.findByRole('option', {
+    name: new RegExp(contest0candidate1.name),
+    selected: true,
+  });
+
+  // Focus should have jumped to the "Next" button because we're using keyboard nav
```

Comment:
```suggestion
  // Focus should have jumped to the "Next" button because we're using PAT nav
```

Reply:
nit: I know this is an old comment, but since we're now distinguishing the different types of keyboard nav, might be more accurate to specify the device

Reply:
ah, thanks for catching!